hack/e2e/run.sh

```diff
@@ -39,15 +39,16 @@ INSTANCE_TYPE=${INSTANCE_TYPE:-c5.large}
 
 AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 USER_ARN=$(aws sts get-caller-identity --query Arn --output text)
-REPO_NAME=${REPO_NAME:-aws-iam-authenticator-e2e}
+REPO_NAME=${REPO_NAME:-aws-iam-authenticator}
 ECR_URL=${AWS_ACCOUNT_ID}.dkr.ecr.${REGION}.amazonaws.com
 IMAGE_NAME=${IMAGE_NAME:-${ECR_URL}/${REPO_NAME}}
 IMAGE_TAG=${IMAGE_TAG:-${TEST_ID}}
 
 K8S_VERSION=${K8S_VERSION:-1.22.10}
 
 KOPS_VERSION=${KOPS_VERSION:-1.23.0}
-KOPS_STATE_BUCKET=${KOPS_STATE_BUCKET:-k8s-kops-auth-e2e}
+KOPS_USERID=$(whoami)
```

Comment:
This might mess up the e2e tests that run in github.  We may need to detect if we are running in github and use `k8s-kops-auth-e2e`, and otherwise use the bucket with `$(whoami)`.

Reply:
this's a good catch. thanks @nckturner. I will fix this